accumulate.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v2"
+	"os"
+	"time"
+)
+
+type AccumulateKey struct {
+	Query     string
+	Zone      string
+	Tenant    string
+	Namespace string
+	Start     time.Time
+}
+
+func (this AccumulateKey) String() string {
+	return this.Query + ":" + this.Zone + ":" + this.Tenant + ":" + this.Namespace
+}
+
+func accumulate(ctx context.Context, date time.Time, cloudscaleClient *cloudscale.Client) (map[AccumulateKey]uint64, error) {
```

Comment:
Without reading the code I have no idea what this function actually does without documentation.
```suggestion
func accumulateBucketMetrics(ctx context.Context, date time.Time, cloudscaleClient *cloudscale.Client) (map[AccumulateKey]uint64, error) {
```

Reply:
I've added some comments explaining what it does.